### ISSUE TITLE
Pharo.image file on bintray is broken

### ISSUE THREAD

**Issue by pavel-krivanek** [guillep/pharo-core#6]
The generated full image file is broken. It shows only the empty white display, does not contain Morphic globals like World and the image size is too small (9.9MB)

https://bintray.com/pharo-project/pharo/Pharo/201701160838-32bit#files

**Comment by guillep** [comment]
How did you test it? Is the entire image that does not respond or just
Morphic?

Is there a log / stack trace?

BTW, we should insert the monkey in the process.

**Comment by pavel-krivanek** [comment]
I simply tried to open the image with pharo-ui. It writes Alive! and can do evaluations from the command line. But Morphic looks like not to be there.
It does not generate any failure output, it seems that the generation of the image failed. On the other hand, in separate jobs based on the build scripts the output image works.

Ad monkey: I already tried to create a monkey job based on bootstrapping from pharo-core git repository. But for the speedup only the bootstrap emulation was used. I would like to look at it closer with you on Tuesday.
https://ci.inria.fr/pharo/view/6.0-Analysis/job/Pharo-6.0-Issue-Validator-with-bootstrap/
